### ISSUE TITLE
random_agent - no such target

### ISSUE THREAD

**Issue by iandanforth** [deepmind/lab#83]
Several of the suggested bazel commands are generating errors like this:

```
idanforth@tower:~/projects/lab$ bazel run :random_agent
ERROR: Skipping ':random_agent': no such target '//:random_agent': target 'random_agent' not declared in package '' defined by /home/idanforth/projects/lab/BUILD
WARNING: Target pattern parsing failed.
ERROR: no such target '//:random_agent': target 'random_agent' not declared in package '' defined by /home/idanforth/projects/lab/BUILD
INFO: Elapsed time: 0.138s
FAILED: Build did NOT complete successfully (0 packages loaded)
ERROR: Build failed. Not running target
```

Build otherwise completes properly and the environment runs perfectly with

```bazel run :game -- --level_script tests/demo_map```

Build Info
commit 038ec1

System Info
Ubuntu 16.04

**Comment by tkoeppe** [comment]
Yes, the README hasn't been updated as part of earlier refactoring. It'll come soon.

**Comment by tkoeppe** [comment]
Fixed by 36c312c628dae782b4f3c669e5e09e85f59774df.
